src/foam/nanos/auth/Phone.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'Phone',
+
+  documentation: 'Phone number information.',
+
+  properties: [
+    {
+      class: 'Boolean',
+      name: 'verified'
+    },
+    {
+      class: 'PhoneNumber',
+      name: 'number',
+      required: true
+    },
+    {
+      class: 'String',
+      name: 'type'
```

Comment:
Define a ChoiceView for this to specify the common types. Or a combobox.

Reply:
Or remove the type if we're going to keep the phone/mobile structure from User.

Reply:
Will do the latter